Makefile.core.mk

```diff
@@ -56,39 +125,151 @@ googleapis_protos = \
+	core/v1/generated.proto \
+	events/v1beta1/generated.proto \
+	extensions/v1beta1/generated.proto \
+	imagepolicy/v1alpha1/generated.proto \
+	networking/v1/generated.proto \
+	networking/v1beta1/generated.proto \
+	node/v1alpha1/generated.proto \
+	node/v1beta1/generated.proto \
+	policy/v1beta1/generated.proto \
+	rbac/v1/generated.proto \
+	rbac/v1alpha1/generated.proto \
+	rbac/v1beta1/generated.proto \
+	scheduling/v1/generated.proto \
+	scheduling/v1alpha1/generated.proto \
+	scheduling/v1beta1/generated.proto \
+	settings/v1alpha1/generated.proto \
+	storage/v1/generated.proto \
+	storage/v1alpha1/generated.proto \
+	storage/v1beta1/generated.proto
+
+k8sapi_packages = \
+	admission/v1 \
+	admission/v1beta1 \
+	admissionregistration/v1 \
+	admissionregistration/v1beta1 \
+	apps/v1 \
+	apps/v1beta1 \
+	apps/v1beta2 \
+	auditregistration/v1alpha1 \
+	authentication/v1 \
+	authentication/v1beta1 \
+	authorization/v1 \
+	authorization/v1beta1 \
+	autoscaling/v1 \
+	autoscaling/v2beta1 \
+	autoscaling/v2beta2 \
+	batch/v1 \
+	batch/v1beta1 \
+	batch/v2alpha1 \
+	certificates/v1beta1 \
+	coordination/v1 \
+	coordination/v1beta1 \
+	core/v1 \
+	events/v1beta1 \
+	extensions/v1beta1 \
+	imagepolicy/v1alpha1 \
+	networking/v1 \
+	networking/v1beta1 \
+	node/v1alpha1 \
+	node/v1beta1 \
+	policy/v1beta1 \
+	rbac/v1 \
+	rbac/v1alpha1 \
+	rbac/v1beta1 \
+	scheduling/v1 \
+	scheduling/v1alpha1 \
+	scheduling/v1beta1 \
+	settings/v1alpha1 \
+	storage/v1 \
+	storage/v1alpha1 \
+	storage/v1beta1
+
+k8sapimachinery_protos = \
+	pkg/runtime/generated.proto \
+	pkg/api/resource/generated.proto \
+	pkg/runtime/schema/generated.proto \
+	pkg/util/intstr/generated.proto \
+	pkg/apis/meta/v1/generated.proto \
+	pkg/apis/meta/v1beta1/generated.proto \
+	pkg/apis/testapigroup/v1/generated.proto
+
+k8sapimachinery_packages = \
+	pkg/runtime \
+	pkg/api/resource \
+	pkg/runtime/schema \
+	pkg/util/intstr \
+	pkg/apis/meta/v1 \
+	pkg/apis/meta/v1beta1 \
+	pkg/apis/testapigroup/v1
 
 all: build
 
+$(googleprotobuf_protos): %:
+	@mkdir -p googleprotobuf/google/protobuf
+	@curl -sS $(GOOGLEPROTOBUF_URL)/$@ -o googleprotobuf/$@
+
+$(googleprotobuf_packages): %: $(googleprotobuf_protos)
+
 $(googleapis_protos): %:
+	@mkdir -p googleapis/google/api googleapis/google/rpc googleapis/google/type
 	@curl -sS $(GOOGLEAPIS_URL)/$@ -o googleapis/$@.tmp
 	@sed -e '/^option go_package/d' googleapis/$@.tmp > googleapis/$@
 	@rm googleapis/$@.tmp
 
 $(googleapis_packages): %: $(googleapis_protos)
-	@$(PROTOC) $(GOGOSLICK_PLUGIN):googleapis -I googleapis googleapis/$@/*.proto
+	@$(PROTOC) $(GOGOSLICK_PLUGIN):googleapis -Igoogleprotobuf -Igoogleapis googleapis/$@/*.proto
 
 $(census_protos): %:
-	@curl -sS $(CENSUS_URL)/$@ -o $@
-	@sed -i.tmp '/^option go_package/d' $@
-	@rm $@.tmp
+	@mkdir -p tmp/opencensus/proto/stats/v1 tmp/opencensus/proto/trace/v1 tmp/opencensus/proto/resource/v1
+	@curl -sS $(CENSUS_URL)/$@ -o tmp/$@.tmp
+	@sed -e '/^option go_package/d' tmp/$@.tmp > tmp/$@
```

Comment:
would an inplace sed work here?

Reply:
Maybe. Just cutting and pasting from previous content...

Reply:
Done.